aequilibrae/matrix/aequilibrae_matrix.py

```diff
@@ -218,7 +233,9 @@ def create_from_omx(
         :return:
         """
 
-        def robust_name(input_name: str, max_length: int, forbiden_names: List[str]) -> str:
+        def robust_name(
```

Comment:
black?

Reply:
It happened in a lot of places. I think I corrected most of them.